corehq/ex-submodules/soil/templates/soil/partials/bootstrap3/download_in_progress.html

```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
```

Comment:
I believed we intentionally dropped dropbox sync from exports... @dimagi/product do you remember that to be the case?